### PR TITLE
[everflow_testbed] extend test to cover egress mirroring

### PR DESCRIPTION
**Pull request by stepanblyschak**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Documentation 

Test according to https://github.com/Azure/SONiC/pull/428

### CODE REVIEW

**Comment by stcheng** [comment]
coudl you resolve the conflicts?

File: ansible/roles/test/files/acstests/everflow_policer_test.py [hunk 1]
```diff
@@ -130,17 +139,23 @@ def checkMirroredFlow(self):
         masked_exp_pkt.set_do_not_care_scapy(scapy.Ether, "dst")
         masked_exp_pkt.set_do_not_care_scapy(scapy.IP, "flags")
         masked_exp_pkt.set_do_not_care_scapy(scapy.IP, "chksum")
+        masked_exp_pkt.set_do_not_care(38*8, len(payload)*8)  # don't match payload, payload will be matched be seperate routine
```
Comment (liat-grozovik) [inline ansible/roles/test/files/acstests/everflow_policer_test.py]: typo? could it be that you wanted to say 'be matched be separated routine?'

File: ansible/roles/test/tasks/everflow_testbed/everflow_main.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  include: "roles/test/tasks/everflow_testbed/apply_config.yml"
+  tags: everflow_tb_configure
+
+- name: Run Everflow tests.
+  include: "roles/test/tasks/everflow_testbed/run_test.yml"
+  vars:
+    dst_port_type: "tor"
+  tags: everflow_tb_test
+
+- name: Run Everflow tests.
```
Comment (liat-grozovik) [inline ansible/roles/test/tasks/everflow_testbed/everflow_main.yml]: suggest to change to 'Run Everflow tests [spine]'

File: ansible/roles/test/tasks/everflow_testbed/everflow_main.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- name: Set flag that recover from config_db.json is needed (default - false)
+  set_fact:
+    recover_from_cfgdb_file: False
+
+- name: Apply Everflow configuration.
+  include: "roles/test/tasks/everflow_testbed/apply_config.yml"
+  tags: everflow_tb_configure
+
+- name: Run Everflow tests.
```
Comment (liat-grozovik) [inline ansible/roles/test/tasks/everflow_testbed/everflow_main.yml]: suggest to change to 'Run Everflow tests [tor]' as there is also another test for spine. special word to differentiate between the runs for debug

File: ansible/roles/test/tasks/everflow_testbed/everflow_main.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  include: "roles/test/tasks/everflow_testbed/run_test.yml"
+  vars:
+    dst_port_type: "spine"
+  tags: everflow_tb_test
+
+- name: Clear Everflow configuration.
+  include: "roles/test/tasks/everflow_testbed/del_config.yml"
+  tags: everflow_tb_cleanup
+
+- name: Reload config
```
Comment (liat-grozovik) [inline ansible/roles/test/tasks/everflow_testbed/everflow_main.yml]: not sure i fully understand. previously it was mentioned that a variable is kept if this need to be done or not but i dont see this variable is being used.

Comment (stepanblyschak) [reply ansible/roles/test/tasks/everflow_testbed/everflow_main.yml]: This task has a ```when: recover_from_cfgdb_file``` condition

**Comment by liat-grozovik** [comment]
@stcheng please review so we can merge

**Pull request merged**